packages/babel-preset-env/test/fixtures/shipped-proposals/node-12/options.json

```diff
@@ -4,5 +4,6 @@
       "targets": { "node": 12 },
       "shippedProposals": true
     }]
-  ]
+  ],
+  "plugins": [["external-helpers", { "helperVersion": "7.100" }]]
```

Comment:
OT: We should really have something like `"helperVersion": "__TEST__CURRENT__"`, and then the test runner replaces `__TEST__CURRENT__` with the current version.
1. This will work across majors
2. There is no risk of actually reaching the fake version we are using here

Reply:
We can also introduce `externalHelpers` option for fixture runner that automatically plug in `external-helpers` with latest `helperVersion`. So we can place `{ externalHelpers: true }` to the root of fixtures folders in most packages which do not care whether the helper is externalized.

Reply:
That's a great idea